lib/elastic_apm/span.rb

```diff
@@ -145,6 +145,9 @@ def set_destination(address: nil, port: nil, service: nil, cloud: nil)
         service: service,
         cloud: cloud
       )
+      context.service = Span::Context::Service.new(
+        target: Span::Context::Service::Target.new(type: context.destination.service.name, name: context.destination.service.type )
```

Comment:
```suggestion
        target: Span::Context::Service::Target.new(name: context.destination.service.name, type: context.destination.service.type )
```

Reply:
I also think that this should have been caught by a test. Can we add one that would cover this part of the code?

Reply:
Oh good catch, and yes, absolutely, adding a test now 👍